api/functions/generate_enums.py

```diff
@@ -2,93 +2,77 @@
+    Column that you would like your enums to be created. The value of the enum is equal to that of the name
+    """
+    app = create_enum_app()
+    db = create_enum_db(app)
 
-	with app.app_context():
-		query = db.session.query(Table).options(load_only(column))
-		rows = db.session.execute(query).fetchall()
+    with app.app_context():
+        query = db.session.query(Table).options(load_only(column))
+        rows = db.session.execute(query).fetchall()
 
-	enum_dict = dict()
+    enum_dict = dict()
 
-	for select in rows:
-		enum_dict.update({select[1]: select[1]})
+    for select in rows:
+        enum_dict.update({select[1]: select[1]})
 
-	if not len(enum_dict):
-		ret_enum = enum.Enum(enum_name, 'EMPTY')
-	else:
-		ret_enum = enum.Enum(enum_name, enum_dict)
+    if not len(enum_dict):
+        ret_enum = enum.Enum(enum_name, 'EMPTY')
+    else:
+        ret_enum = enum.Enum(enum_name, enum_dict)
 
-	return graphene.Enum.from_enum(ret_enum)
+    return graphene.Enum.from_enum(ret_enum)
 
 
 def create_enum_with_descriptions(Table, enum_name, enum_column, description_column):
-	"""
-	Function to allow the creation of enums with descriptions for various uses. Function takes in SQLAlchemy Model,
-	and the Column that you would like your enums to be created. The value of the enum is equal to that of the name
-	"""
-	app = create_enum_app()
-	db = create_enum_db(app)
+    """
+    Function to allow the creation of enums with descriptions for various uses. Function takes in SQLAlchemy Model,
+    and the Column that you would like your enums to be created. The value of the enum is equal to that of the name
+    """
 
-	with app.app_context():
-		query = db.session.query(Table).options(load_only(enum_column))
-		rows = db.session.execute(query).fetchall()
+    app = create_enum_app()
+    db = create_enum_db(app)
 
-	enum_dict = dict()
+    with app.app_context():
+        query = db.session.query(Table).options(load_only(enum_column))
+        rows = db.session.execute(query).fetchall()
 
-	for select in rows:
-		enum_dict.update({select[1]: select[1]})
+    enum_dict = dict()
 
-	if not len(enum_dict):
-		ret_enum = enum.Enum(enum_name, 'EMPTY')
-	else:
-		ret_enum = enum.Enum(enum_name, enum_dict)
+    for select in rows:
+        enum_dict.update({select[1]: select[1]})
 
-	description_dict = dict()
+    if not len(enum_dict):
+        ret_enum = enum.Enum(enum_name, 'EMPTY')
+    else:
+        ret_enum = enum.Enum(enum_name, enum_dict)
 
-	for e in ret_enum:
-		with app.app_context():
-			query = db.session.query(Table).filter(getattr(Table, enum_column) == e.name).options(load_only(description_column))
-			row = db.session.execute(query).first()
+    description_dict = dict()
 
-			try:
-				description_dict.update({e.name: row[1]})
-			except:
-				description_dict.update({e.name: "Error loading description"})
+    for e in ret_enum:
```

Comment:
single letter variables like this are a dangerous game.  I assume it stands for 'enum' but it may be worth making it explicit.